defaults/main.yml

```diff
@@ -141,12 +141,17 @@ netdata_stream_enabled: false
 # Defines location of Netdata stream configuration file
 netdata_stream_config_file: /etc/netdata/stream.conf
 
-# Defines Netdata API Key (must be generated with command uuidgen)
+# Defines Netdata API Keys (must be generated with command uuidgen)
 netdata_stream_api_key: 11111111-2222-3333-4444-555555555555
+netdata_strem_send_api_key: "{{ netdata_stream_api_key }}"
```

Comment:
Typo in variable

Reply:
Thanks for your review and your catch! Fixed